geopandas/base.py

```diff
@@ -295,12 +291,10 @@ def length(self):
 
     @property
     def is_valid(self):
-        """Returns a ``Series`` of ``dtype('bool')`` with value ``True`` for
-        geometries that are valid.
+        """Return if geometries are valid.
```

Comment:
This feels a bit like a regression in the docstring. I understand that the original may be too long for strict pydocstyle but it is surely more explicit. Though it is nothing I couldn't live with.

Reply:
There is indeed information being removed, but I think it is information that is now in the wrong place: it should be in the type annotations and in the "Return" section of the docstring rather than in the first line of the docstring.

Reply:
Fair.

Reply:
Yes, I think indeed the return section should be added...